### DISCUSSION TITLE
Would anyone like to add Mojo to GitHub Linguist?

### DISCUSSION THREAD

**Discussion by alecgargett** [modular/modular#1228] [Ideas]
Mojo appears to qualify to be added GitHub Linguist under the new criteria listed here:
https://github.com/github-linguist/linguist/issues/5756

The instructions on how to do so are here:
https://github.com/github-linguist/linguist/blob/master/CONTRIBUTING.md#adding-a-language

Once it has been added it will automatically appear in other projects that use GitHub Linguist such as Languish:
https://tjpalmer.github.io/languish/

I'd love to be able to track Mojo's growth over time using Languish and it'd be nice for others to see Mojo listed there too.

I'm not quite qualified to add Mojo myself so I'd greatly appreciate if someone else with the knowhow were to do it.

**Comment by alecgargett** [comment]
My mistake. it looks it only has 1.2K results and it needs 2K:

https://github.com/search?q=path%3A*.mojo&type=Code&ref=advsearch&l=&l=

I guess I'll re-open this discussion when it reaches 2K

https://github.com/search?q=path%3A*.mojo&type=Code&ref=advsearch&l=&l=

**Comment by dom96** [comment]
See https://github.com/github-linguist/linguist/pull/6400

**Reply by alecgargett** [reply]
Thank you. I should have thought to search there instead of just here.
